tests/zkevm/test_worst_compute.py

```diff
@@ -952,6 +952,49 @@ def test_worst_mod(
     )
 
 
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize("opcode", [Op.MLOAD, Op.MSTORE, Op.MSTORE8])
+@pytest.mark.parametrize("offset", [0, 1, 31])
+@pytest.mark.parametrize("offset_initialized", [True, False])
+@pytest.mark.parametrize("big_memory_expansion", [True, False])
```

Comment:
To indicate if the memory has to be expanded in the setup phase, so the memory size isn't minimal.